src/Plugin/migrate/process/Pipeline.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateException;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use Drupal\oe_migration\Entity\MigrationProcessPipeline;
+
+/**
+ * This plugin executes a process pipeline loaded from configuration.
+ *
+ * This plugin is useful to avoid duplicating process definitions in Yaml.
+ * You may have a suite of process plugins you are using in several places or a
+ * single process plugin with want to duplicate complex/specific configuration
+ * that you don't want to duplicate.
+ *
+ *  Available configuration keys:
+ *   - id: Pipeline identification. The ID of the pipeline that you want to use.
+ *   - placeholders (optional): an array of placeholders to replace in the
+ *     pipeline definition. Useful to increase the possibility of pipeline
+ *     reuse.
+ *
+ * @codingStandardsIgnoreStart
+ * Usage example:
+ * @code
+ * process:
+ *   'body/value':
+ *     - plugin: oe_migration_pipeline
+ *       id: my_process_pipeline
+ *       source: 'body/0/value'
+ *       placeholders:
+ *         TOKEN1: value_token1
+ * @endcode
+ *
+ * The above example expects a process pipeline "my_process_pipeline" defined
+ * in configuration.
+ *
+ * Create a "oe_migration.oe_migration_process_pipeline.my_process_pipeline.yml"
```

Comment:
This bit of comment is not helpful/does not belong here. Should go maybe into the readme. Just creating that file won't do anything. Will need to be imported and created as a config entity. Consider making a form for creating/editing process pipelines where the field for storing the definitions is a simple YAML textfield that gets stored as is? This will increase the usability of this system.

Reply:
I'm not sure how to manage it. I think that the UI is not very useful (at least for us) and even for Migrate itself as a modules suite. All complex migrate process are managed usually with files, drush, etc. and I'm not sure that the UI will increase the usability in this case. But you're rigth, this description not very helpful and you have to create the file and import it, something that it's not very intuitive maybe.

Mmm... should we improve the documentation? Or we have to spend time in create a form to load a yaml (it's not a lot of time, it's a simple form, but it's time) with the risk that nobody use it? I think that we can take the first approach, I'm going to improve the documentation in the README related with this plugin and if in the future we see that it has sense to create a form, we will developed it.

Reply:
Ok, I added some info to the README. It includes a way to import the configuration file using the UI and the Configuration Manager module (included in Core). I added to some tips to give clues to the oe_migrations users in how to import the file in different ways (using drush, adding in the install directory or, as I mentioned, using the UI). Maybe we can iterate over this doc or where can we place it but I think that this approach is enough for the moment. We always can improve it in the future.

Reply:
OK, I guess that's fine.